upstream.wrap

```diff
@@ -1,7 +1,6 @@
 [wrap-file]
-directory = librdkafka-1.5.0
-
-source_url = https://github.com/edenhill/librdkafka/archive/v1.5.0.tar.gz
-source_filename = rdkafka-1.5.0.tgz
-source_hash = f7fee59fdbf1286ec23ef0b35b2dfb41031c8727c90ced6435b8cf576f23a656
+directory = librdkafka-1.5.2
 
+source_url = https://github.com/edenhill/librdkafka/archive/v1.5.2.tar.gz
+source_filename = rdkafka-1.5.2.tgz
+source_hash = ca3db90d04ef81ca791e55e9eed67e004b547b7adedf11df6c24ac377d4840c6 
```

Comment:
Can you add a [provide] section: https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section

This should map both pkg-config dependency names to their dependency variables.